test/test_helper.js

```diff
@@ -0,0 +1,21 @@
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import sinon from 'sinon';
+import jsdom from 'jsdom';
+
+const html = "<!doctype html><html><head><meta charset='utf-8'></head><body></body></html>";
```

Comment:
Line is too long.<br>'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>Mixed double and single quotes.